scipy/stats/_sensitivity_analysis.py

```diff
@@ -26,6 +24,12 @@
 ]
 
 
+class PPFDist(Protocol):
+    @property
+    def ppf(self) -> Callable[..., np.float64]:
+        ...
```

Comment:
definition of protocol is moved above where it is used. The change from `float` to `np.float64` comes from scipy-stubs (while we are here I guess)

Reply:
Yea the `float` wasn't correct on `numpy<2.2`, because `np.float64` wasn't annotated as a subtype of `float` (I fixed it for the upcoming `numpy>=2.2` release though)

Reply:
Oh I just realized that this is (or at least should) be equivalent to

```py
class PPFDist(Protocol):
    def ppf(self, /, *args: Any, **kwargs: Any) -> np.float64: ...
```

although... maybe it's actually more like

```py
class PPFDist(Protocol):
    @staticmethod
    def ppf(*args: Any, **kwargs: Any) -> np.float64: ...
```

🤔

Reply:
Oh because of Liskov's substitution principle, I believe you could also remove the `*args: Any, **kwargs: Any` (i.e. function parameters are contravariant) like

```py
class PPFDist(Protocol):
    def ppf(self, /) -> np.float64: ...
```

or

```py
class PPFDist(Protocol):
    @staticmethod
    def ppf() -> np.float64: ...
```

(still not sure about the difference between those two, but I expect both to work just fine in this case)

Reply:
I guess we can leave those improvements to be made in scipy-stubs?

Reply:
> I guess we can leave those improvements to be made in scipy-stubs?

Sure, that's fine by me.

But then I'd suggest removing the `Protocol` altogether. Not because this is incorrect, but because if it's not going to be properly maintained or even validated, and because `scipy-stubs` provides a better alternative, then there's no reason to keep it.

Reply:
Okay, removed now